ocaml/xapi/message_forwarding.ml

```diff
@@ -3851,5 +3851,9 @@ module Forward = functor(Local: Custom_actions.CUSTOM_ACTIONS) -> struct
 
 	module VGPU_type = struct end
 	module LVHD = struct end
+
+	module PVS_farm = struct
```

Comment:
Wasn't this, and a similar section in api_server.ml, needed in the previous patch to make things compile?